src/components/Trans.tsx

```diff
@@ -80,14 +87,11 @@ const Trans = ({
   ...props
 }: {
   i18nKey: string;
-  [key: string]: any;
+  [key: string]: React.ReactNode | React.FC<any>;
 }) => {
   const { t } = useI18n();
 
-  return React.createElement.apply(
-    React.createElement,
-    [React.Fragment, {}].concat(getTransChildren(t(i18nKey), props)) as any,
-  );
+  return <>{getTransChildren(t(i18nKey), props)}</>;
```

Comment:
@ad1992 the reason for using `createElement` is that React complains if the `children` don't have the `key` prop.

`<>{[child1, child2]}</>` is translated to `createElement(React.Fragment, { children: [child1, child2] })` and if `child1` or `child2` don't have `key` prop, React will complain.

`<>{child1}{child2}</>` is translated to `createElement(React.Fragment, {}, child1, child2)` (which is what we want to do).

Reply:
There is now a warning when running tests:
```
  console.error
    Warning: Each child in a list should have a unique "key" prop.

    Check the render method of `Trans`. See https://reactjs.org/link/warning-keys for more information.
        at a
        at Trans (.../excalidraw/src/components/Trans.tsx:85:3)
        at div

      92 |   const { t } = useI18n();
      93 |
    > 94 |   return <>{getTransChildren(t(i18nKey), props)}</>;
         |          ^
      95 | };
      96 |
      97 | export default Trans;
```

Reply:
The following is cleaner than my original code. My original code is quite old when rest arguments did not yet exist.

```js
return React.createElement(
    React.Fragment,
    {},
    ...getTransChildren(t(i18nKey), props),
  );
```

Reply:
yep was gonna commit this only :p